### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by fabiencastan**

### CODE REVIEW

File: buttleofx/gui/Tab.qml [hunk 1]
```diff
@@ -84,6 +85,6 @@ Item {
 
                 onClicked: fullscreenClicked(true)
             }
-        }//tab
-    }//Row
-}//tabBar
+        } // Tab
```
Comment (fabiencastan) [inline buttleofx/gui/Tab.qml]: We may remove this kind of comments, no?
May not be maintained properly... so I prefer no information than a wrong one.

File: buttleofx/gui/graph/graphWrapper.py [hunk 2]
```diff
@@ -81,49 +55,25 @@ def deleteNodeWrapper(self, nodeName):
...
+                if nodeWrapper.getNbInput() > 0 :
                     clipConnected_input = self.getConnectedClipWrapper(nodeWrapper.getSrcClips().get(0), False)
-                else :
+                else:
                     clipConnected_input = None
+
                 clipConnected_output = self.getConnectedClipWrapper_Output(nodeWrapper.getOutputClip())
-                if(clipConnected_input and clipConnected_output):
+
+                if (clipConnected_input and clipConnected_output):
```
Comment (fabiencastan) [inline buttleofx/gui/graph/graphWrapper.py]: We can remove parenthesis in python.

File: buttleofx/gui/graph/graphWrapper.py [hunk 3]
```diff
@@ -183,46 +119,33 @@ def getNodeWrapperByIndex(self, nodeIndex):
...
+
     @QtCore.pyqtSlot(QtCore.QObject, bool, result=QtCore.QObject)
     def getConnectedClipWrapper(self, clipWrapper, disable):
         """
             Returns the clip connected to an input clip if it exists.
         """
         for connection in self._connectionWrappers:
-            if((clipWrapper.getNodeName() == connection.getIn_clipNodeName() and clipWrapper.getClipName() == connection.getIn_clipName())):
-                if(disable):
+            if (clipWrapper.getNodeName() == connection.getIn_clipNodeName() \
```
Comment (fabiencastan) [inline buttleofx/gui/graph/graphWrapper.py]: Don't need "\" at the end of the line.
That's the goal of parenthesis.

File: buttleofx/gui/graph/qml/Graph.qml [hunk 4]
```diff
@@ -136,19 +132,19 @@ Rectangle {
...
-            acceptDrop = hasUrls
-        }
-        onDrop: {
-            console.log("Drop external files:", acceptDrop)
-            if(acceptDrop) {
-                _buttleManager.nodeManager.dropFile(firstUrl, pos.x - m.graphRoot.originX, pos.y - m.graphRoot.originY)
-            }
-        }
-    }
+      ExternDropArea {
```
Comment (fabiencastan) [inline buttleofx/gui/graph/qml/Graph.qml]: Is there a reason to remove indentation in commented code?

File: buttleofx/gui/paramEditor/qml/ParamBoolean.qml [hunk 5]
```diff
@@ -69,12 +70,13 @@ Item {
...
 
-            MouseArea{
+            MouseArea {
                 anchors.fill: parent
                 acceptedButtons: Qt.LeftButton
+
                 onPressed: {
                     paramObject.value = (paramObject.value == false) ? true : false
-                    paramObject.pushValue(paramObject.value);           
+                    paramObject.pushValue(paramObject.value);
```
Comment (fabiencastan) [inline buttleofx/gui/paramEditor/qml/ParamBoolean.qml]: no ; needed at the end of lines

**Comment by fabiencastan** [comment]
Really nice work!

File: buttleofx/gui/Tab.qml (hunk 1 above)
Comment (JamesWrigley) [inline buttleofx/gui/Tab.qml]: Yeah, good point. Will remove all those.

File: buttleofx/gui/graph/graphWrapper.py (hunk 2 above)
Comment (JamesWrigley) [inline buttleofx/gui/graph/graphWrapper.py]: True, will fix that.

File: buttleofx/gui/graph/graphWrapper.py (hunk 3 above)
Comment (JamesWrigley) [inline buttleofx/gui/graph/graphWrapper.py]: Ah I see, will fix.

File: buttleofx/gui/graph/qml/Graph.qml (hunk 4 above)
Comment (JamesWrigley) [inline buttleofx/gui/graph/qml/Graph.qml]: Not really, it's just Emac's default indentation style. I'll re-indent it.

File: buttleofx/gui/paramEditor/qml/ParamBoolean.qml (hunk 5 above)
Comment (JamesWrigley) [inline buttleofx/gui/paramEditor/qml/ParamBoolean.qml]: Ah, will remove those then.

**Comment by JamesWrigley** [comment]
Thanks :)

**Comment by fabiencastan** [comment]
This will continue to be enhanced in the future, but this step can be merged into develop, no?

**Comment by JamesWrigley** [comment]
Yep, sure.

**Pull request merged**